handbook/engineering/search/index.md

```diff
@@ -39,6 +41,7 @@ Our current focus is documented in the [tracking issue for the current milestone
   - **Plan:** Streaming search
   - **Owner:** Keegan
   - **Status:** In progress
+  - **Estimated completion:** TODO
```

Comment:
@keegancsmith could you add an (even rough) estimated completion milestone for this work?

Reply:
```suggestion
  - **Estimated completion:** 3.21
```